src/awkward/operations/ak_to_layout.py

```diff
@@ -194,16 +193,6 @@ def _impl(
         return _handle_array_like(
             obj, promoted_layout, primitive_policy=primitive_policy
         )
-    elif Jax.is_own_array(obj):
```

Comment:
this doesn't have to go either, see my comment related to `from_jax`

Reply:
done.